python/JupyROOT/CMakeLists.txt

```diff
@@ -1,17 +1,18 @@
 if (ROOT_pyroot_FOUND AND NOT ROOT_CLASSIC_BUILD) # Do not run with classic build
 
 set(MODULES_LOCATION ${ROOTSYS}/lib/JupyROOT/helpers)
-set(NBDIFFUTIL ${CMAKE_CURRENT_SOURCE_DIR}/nbdiff.py )
+set(NBDIFFUTIL ${CMAKE_CURRENT_SOURCE_DIR}/nbdiff.py)
```

Comment:
Changes in this file are superseded by:

https://github.com/root-project/roottest/pull/511

Please remove this commit from the PR, thanks @linev !